R/resample-third-party-hazard-maps-to-zillow-grid.R

```diff
@@ -0,0 +1,22 @@
+# Resample the Fire risk map to be at the same resolution/extent as the Zillow data
+
+library(tidyverse)
+library(raster)
+library(rasterVis)
+
+template <- raster::raster("data/EmptyGrid/Empty_250_US.tif")
+
+# Source: https://www.fs.usda.gov/rds/archive/catalog/RDS-2016-0034
```

Comment:
Instead of assuming users will go do this URL and download a file manually, it would be more convenient if the script handled the download/unzipping if the expected local files don't exist

Reply:
Ah, this brings up a question I had after your intro to reproducible science presentation. Where do we draw the line here? I was under the impression that it would be premature to hard code a download/unzip step before putting the data in a permanent location stamped with a DOI. Until the data are put into an archive-level repository, there will always be an extra manual step (whether that's the user downloading the data or the coder updating the link every time it gets moved to a different temporary home online)? 

Is the best practice for the coder to keep the download link up to date?

Reply:
For now we can use whatever URLs point to the data we're using to avoid manual steps in the short run. That will make it easier for us to iterate on stuff, but long-term, yes we might put data in a more permanent location.

Reply:
That makes sense! Aside from the individual scripts to configure/standardize each raster hazard layer, there is also now a separate script that grabs the configured versions of each hazard layer from the Google drive. Really, this is the only script that needs to be run in order to work with the data while the individual "configure-['hazard name']" scripts go through a custom process per hazard to get them on the zillow CRS and extent and to standardize the 0/NA-handling.